code/src/App.js

```diff
@@ -1,12 +1,51 @@
 import React from 'react'
 import data from './data.json'
+import { Album } from './Album'
+import { Heading } from './Heading'
+import { Singles } from './Singles'
 
 console.log(data)
```

Comment:
again logs. You really always need to remove it from the finished version of your code